### DISCUSSION TITLE
Regression logisitique (évènement rare)

### DISCUSSION THREAD

**Discussion by egemirf** [larmarange/guide-R#58] [Forum]
Bonjour Monsieur
Merci beaucoup pour les cours.
J'ai une question.
La regression logistique binaire multiple nécessite normalement au moins 5 à 10 évènements par nombre de variables dans le modème.
Je suis dans une situation où je recherche les facteurs associés au décès chez les patients tuberculeux. La taille d'échantillon est de 201 et il y a 13 décès. Je suis vraiment obligé de mettre maximum deux variables dans mon modèle ? N'y a t-il pas des méthodes pour contourner ce problème?
Merci beaucoup

**Comment by larmarange** [comment]
Dans tous les cas, vous n'aurez que peu de puissance.

La première étape est déjà de réaliser des analyses bivariées univariables et déjà voir si en analyse bivariée (comparaison de proportions) vous avez des variables avec un effet significatif. Vous pouvez ensuite démarrer un modèle logistique avec les variables ayant un effet au moins au seuil de 20%, puis essayer de réduire votre modèle (par exemple avec sélection pas à pas descendante, voir fonction step() ) afin de ne garder que les effets les plus importants.

**Reply by egemirf** [reply]
Merci beaucoup pour votre réponse et pour la rapidité. 
J'aurais entendu parler de la pénalisation de coefficient par la regression de Lasso. Est -elle adaptée dans mon cas?

**Comment by larmarange** [comment]
Je ne l'ai jamais essayé et donc je suis mal placé pour en parler. A tenter
éventuellement

Joseph LARMARANGE

**Reply by egemirf** [reply]
D'accord. Merci beaucoup
